doc/source/user_guide/io.rst

```diff
@@ -3924,6 +3924,20 @@ The look and feel of Excel worksheets created from pandas can be modified using
 * ``float_format`` : Format string for floating point numbers (default ``None``).
 * ``freeze_panes`` : A tuple of two integers representing the bottommost row and rightmost column to freeze. Each of these parameters is one-based, so (1, 1) will freeze the first row and first column (default ``None``).
 
+.. note::
+
+    As of Pandas 3.0, by default spreadsheets created with the ``to_excel`` method
+    will not contain any styling. Users wishing to bold text, add bordered styles,
+    etc in a worksheet output by ``to_excel`` can do so by using ``Styler.to_excel``
+    to create styled excel files.
+
+
+.. code-block:: python
+
+    css = "border: 1pt solid #111222"
+    styler = df.style.map(lambda x: css)
```

Comment:
I don't think this is equivalent to how Excel files are currently styled, is that right? Would it make sense to do that instead?

Reply:
I agree, think we need to provide the exact replication. Typing on phone now but I suggest the its probably going to be something like;

```python
css = "border: 1px solid black; font-weight: bold;"
# instead of df.to_excel("myfile.xlsx")
df.style.map_index(lambda x: css).map_index(lambda x: css, axis=1).to_excel("myfile.xlsx")
```

Reply:
@attack68 Yeah doing a quick comparison of the above with a basic call to `DataFrame.to_excel()` shows that the above styling is the same as the current default styling.